index.html

```diff
@@ -1,14 +1,17 @@
 <html>
-  <body>
-    <p id='server-time'></p>
+  <head>
     <script>
-      var HOST = location.origin.replace(/^http/, 'ws')
-      var ws = new WebSocket(HOST);
-      var el = document.getElementById('server-time');
+    var HOST = location.origin.replace(/^http/, 'ws')
```

Comment:
While you're here, you might want to update to modern JS with `let` and arrow functions